pipeline/metadata/satellite.py

```diff
@@ -341,8 +341,7 @@ def _key_by_date_domain(row: Row) -> Tuple[DateDomainKey, Row]:
 def _partition_test_and_control(row_tuple: Tuple[DateDomainKey, Row],
                                 _: int) -> int:
   row: Row = row_tuple[1]
-  # 'anomaly' is None for control measurements
```

Comment:
Maybe add a todo to make control a separate pcollection.

Reply:
I think we actually don't want to do this. Controls and non-control tests are the same format/schema (by design). And we want to do the same operations on them (adding resolver and answer metadata). The only thing we could buy by splitting them out earlier (I'm assuming you mean in the initial filanema partitioning function) is having to duplicate all those steps twice in our pipeline graph. Currently by only partitioning them just before we need them we're able to keep the total graph simpler.